src/main/java/rx/util/async/Async.java

```diff
@@ -92,17 +187,17 @@ private Async() {
     }
     
     /**
-     * Convert a synchronous function call into an asynchronous function call through an Observable.
+     * Convert a synchronous callable call into an asynchronous function call through an Observable.
      * <p>
      * <img width="640" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/toAsync.png" alt="">
      *
      * @param <R> the result value type
-     * @param func the function to convert
+     * @param func the callable to convert
      * @return a function that returns an Observable that executes the {@code func} and emits its returned value
      * @see <a href="https://github.com/ReactiveX/RxJava/wiki/Async-Operators#wiki-toasync-or-asyncaction-or-asyncfunc">RxJava Wiki: toAsync()</a>
      * @see <a href="http://msdn.microsoft.com/en-us/library/hh229182.aspx">MSDN: Observable.ToAsync</a>
      */
-    public static <R> Func0<Observable<R>> toAsync(Func0<? extends R> func) {
+    public static <R> Func0<Observable<R>> toAsync(Callable<R> func) {
```

Comment:
The thought was that `Func0` extends `Callable`, so I thought it should be fine. I could revert this change and add a second method `toAsyncThrowing(Callable)` if this does indeed cause a problem.